std/typecons.d

```diff
@@ -8653,6 +8653,15 @@ replacement.
 */
 template ReplaceType(From, To, T...)
 {
+    template isAliasThis(T, U)
+    {
+        static foreach (at; __traits(getAliasThis, T))
+            static if (is(typeof(__traits(getMember, T, at)) : U))
+                enum tmp = true;
+
+        enum isAliasThis =__traits(compiles, assert(tmp));
```

Comment:
Is this the best way to do this? I suppose it does doesn't really matter if it is made private.

Reply:
Probably not. This would be better:
```
    template isAliasThis(T, U) {
        enum checkAlias(string name) = is(typeof(__traits(getMember, T, name)) : U);
        enum isAliasThis = anySatisfy!(checkAlias, __traits(getAliasThis, T));
    }
```